src/Builder.jl

```diff
@@ -105,9 +107,20 @@ function Selectors.runner(::Type{SetupBuildDirectory}, doc::Documents.Document)
         for file in files
             src = normpath(joinpath(root, file))
             dst = normpath(joinpath(build, relpath(root, source), file))
+
+            if workdir == :build
+                # set working directory to be the same as `build`
+                wd = normpath(joinpath(build, relpath(root, source)))
+            elseif typeof(workdir) <: Symbol
+                # Maybe allow `:src` and `:root` as well?
+                throw(ArgumentError("Unrecognized working directory option '$workdir'"))
+            else
+                wd = normpath(joinpath(root, workdir))
```

Comment:
```suggestion
                wd = normpath(joinpath(doc.user.root, workdir))
```
instead?

At the moment, relative paths become relative to the directory containing the `.md` file. I don't think that's intended? I.e. currently, if you do `workdir="."` in `make.jl`, it is equivalent to `:build`, while I'd expect it to be `workdir = @__DIR__`.

Somewhat related: how do you feel about `workdir = @__DIR__` or `workdir = joinpath(@__DIR__, "..")` being the recommended ways to set the path an absolute path for `workdir`?

Reply:
Do I understand correctly that this would mean the path is no longer relative to `root`, but we're recommending people set the absolute path themselves? And is `@__DIR__` the directory in the `build` folder where it ends up, or the folder of the `.md` file in `src/`?

Reply:
Depends on what you mean by "`root`". With my change, the working directory will always be set to a fixed path relative to `makedocs`' `root` argument.

Currently it is relative to `dirname("path/to/page.md")? So e.g. if `workdir="foo"`, then Documenter would `cd` into `build/foo` for `src/page.md`, but into `build/subdir/foo` for `src/subdir/page.md`?